.github/workflows/bash_unit_testing.yml

```diff
@@ -1,7 +1,6 @@
 name: bash Unit Testing
 
 on:
-  push:
```

Comment:
Does removing this mean the action only runs once when the pull request is created?